README.md

```diff
@@ -96,6 +96,10 @@ Next, add this to your crate:
 extern crate crossbeam;
 ```
 
+## Compatibility
+
+The minimum supported Rust version is 1.28. Any change to this is considered a breaking change.
```

Comment:
Thank you for writing this up! I think we *really* should bring Crossbeam to 1.0 soon. I will follow up with an issue on what changes we need to do to get there.

Reply:
One additional thing I've noticed: the umbrella crate `crossbeam` did not made a breaking change, it just bumped major versions of it's public deps, which is, dejure, a  breaking change (which doesn't mean it creates problems in practice, only bug reports will tell :) ).

Reply:
I will admit it: this was a very rushed and belated release.

Part of the reason is the fact that I *dread* making releases because they're so time-consuming and finicky (update readmes, changelogs, and `Cargo.toml`s for every crate, fix up dependencies, publish to crates.io, publish git tags) and I *always* make mistakes while releasing new versions. I don't think there's been a single release that went smoothly without mistakes.

Grouping all (or most) crossbeam crates into the main crate (kind of like what `tokio` is doing these days) and publishing 1.0 would alleviate this pain for everyone.